packages/terra-consumer-nav/src/components/help-button/HelpButton.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../variables';
+
+:local {
+  .help-button {
+    background: var(--terra-consumer--help-button-background, #fff);
+    border-radius: 20px;
+    bottom: 15px;
+    box-shadow: 0 1px 1px #666;
+    padding: 10px;
+    position: absolute;
```

Comment:
is this positioned absolutely or fixed? I think it should be fixed to the bottom right shouldn't it?